utils/validation.js

```diff
@@ -419,6 +446,32 @@ const checkDerivedVariables = async (token, siteCode) => {
         updates = { ...updates, ...sampleUpdates};
     }
 
+    if(anyRefusalWithdrawal) {
+
+        const refusedBaselineBlood = data[refusalWithdrawalConcepts.refusedBaselineBlood] === 353358909;
+        const refusedBaselineSpecimenSurvey = data[refusalWithdrawalConcepts.refusedBaselineSpecimenSurvey] === 353358909;
+        const refusedBaselineSaliva = data[refusalWithdrawalConcepts.refusedBaselineSaliva] === 353358909;
+        const refusedFutureSamples = data[refusalWithdrawalConcepts.refusedrefusedFutureSamplesBaselineBlood] === 353358909;
+        const refusedFutureSurveys = data[refusalWithdrawalConcepts.refusedFutureSurveys] === 353358909;
+        const refusedBaselineUrine = data[refusalWithdrawalConcepts.refusedBaselineUrine] === 353358909;
+        const refusedBaselineSurveys = data[refusalWithdrawalConcepts.refusedBaselineSurveys] === 353358909;
+        const suspendedContact = data[refusalWithdrawalConcepts.suspendedContact] === 353358909;
+        const withdrewConsent = data[refusalWithdrawalConcepts.withdrewConsent] === 353358909;
+        const revokeHIPAA = data[refusalWithdrawalConcepts.revokeHIPAA] === 353358909;
+        const dataDestroyed = data[refusalWithdrawalConcepts.dataDestroyed] === 353358909;
+        const refusedFutureActivities = data[refusalWithdrawalConcepts.refusedFutureActivities] === 353358909;
+        const deceased = data[refusalWithdrawalConcepts.deceased] === 353358909;
+
+        const refusalUpdates = {
+            '451953807':    refusedBaselineBlood || refusedBaselineSpecimenSurvey || refusedBaselineSaliva || refusedFutureSamples ||
+                            refusedFutureSurveys || refusedBaselineUrine || refusedBaselineSurveys || suspendedContact ||
+                            withdrewConsent || revokeHIPAA || dataDestroyed || refusedFutureActivities ||
+                            deceased ? 353358909 : 104430631
+        }
+
+        updates = { ...updates, ...refusalUpdates};
+    }
+
     console.log("UPDATES");
```

Comment:
console logs left intentionally?

Reply:
yes, this is there for debugging to show what was actually updated each time this function is ran for a participant